packages/story-editor/src/components/panels/design/textStyle/karma/stylePresets.karma.js

```diff
@@ -65,8 +65,8 @@ describe('Panel: Style Presets', () => {
       });
 
       // Verify that no styles are added currently.
-      const noStylesText = fixture.screen.getByText('No Styles Saved');
-      expect(noStylesText).toBeDefined();
+      const noStylesText = fixture.screen.getAllByText('No Styles Saved');
+      expect(noStylesText.length).toBeGreaterThanOrEqual(1);
```

Comment:
Can there be more than one?

Reply:
Ah, it is from the two different panels - because the text exists in both the text insertion panel and the design panel?

Perhaps it should have a test for each separately, to confirm that it exists in both panels individually. That will also make the test (correctly) fail once the panels are merged in #10111.

Reply:
This test should actually really test for the one in the Library. Adjusted to do that.